scripts/preprocess_markdown_includes.py

```diff
@@ -0,0 +1,35 @@
+import re
```

Comment:
Include a note that if you're running/checking this manually, you need to ensure you're in the root folder of your project.